INSTALL.Unix.md

```diff
@@ -66,7 +66,7 @@ You can install the dependencies by running:
         build-essential pkg-config erlang erlang-reltool \
         libicu-dev libmozjs-60-dev python3
 
-Your distribution may have `libmozjs-68-dev` instead of 60. Both are supported.
+Your distribution may have `libmozjs-68-dev` instead of 60. Both are supported. Newer versions of `libmozjs-...-dev` may also work; you can find which are available using `apt-cache search libmozjs`.
```

Comment:
I would suggest this, which is i think a bit more comprehensive approach.
`
Your distribution might provide libmozjs-68-dev instead of libmozjs-60-dev, and both are supported. Additionally, newer versions of libmozjs-...-dev may also be compatible. To see which versions are available, use the command "apt-cache search libmozjs"
`